### PR TITLE
Clean up the THICK log messages

### PR DESCRIPTION
**Pull request by spikeekips**
### Background

Just look at these log messages,

```
{"caller":"node_runner.go:484", "error":{"code":142,"message":"invalid voting
basis", "data":{}},"lvl":"dbug", "message":"{"H":{"version":"1",
"hash":"46k1bWaNLpaExTd3gVUdUjW5jVB9LTBay8AkkmuzFNn6",
"signature":"3pqynbyd74F9Z8BsawuEqxVG2YEn39RTs9F3wAFJ6RcL2eTKCL2Rx1sPfew9r97m7QCa7sKebVHbpDhDq3CBK9U2",
"proposer_signature":"5XAZeTDZCFTmDkpskn4L2mjiGFzL88co5H9zAPivJdx21ubx3E6ExTH2Uh1dGFNGJyhyDjUr8fM7RzAHjD9CfnY8"},"B":{"confirmed":"2018-11-23T00:57:33.372076680Z",
"proposed":{"confirmed":"2018-11-23T00:57:33.354288922Z",
"proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"voting_basis":{"round":0,"height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":2},"transactions":null,"proposer_transaction":{"H":{"version":"1",
"created":"2018-11-23T00:57:33.353805821Z",
"signature":"bTGdFEHjWvA6ZpmX89ERn1txZg3y5PxizeWBKusHomxgmi1awrmwjZv3rLeLhemsmTK5LU7WjDdKMpPtxKfqQnz"},"B":{"source":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"fee":"0",
"sequence_id":0,"operations":[{"H":{"type":"collect-tx-fee"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"0",
"txs":0,"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}},{"H":{"type":"inflation"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"100000000000", "initial_balance":"1000000000000000000",
"ratio":"0.00000010000000000",
"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}}]}}},"source":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"state":3,"vote":"YES", "reason":null}}", "module":"noderunner", "msg":"failed
to handle message", "node":"GARA.FANE", "t":"2018-11-23T00:57:33.399737603Z"}
```

```
{"ballot":{"H":{"version":"1",
"hash":"59SgKRqPb2nHcqY94orcNF2MxDpPkWQhGe2SmoPy7h1f",
"signature":"4MGodCSz4W9fF926w6TBhdD8xEYiVsCKGgak7EXzWt6YCFSJnZQwwopcdQ6pCdqBaVkGrUBePkBfJ4cpX8bcHXK7",
"proposer_signature":"5XAZeTDZCFTmDkpskn4L2mjiGFzL88co5H9zAPivJdx21ubx3E6ExTH2Uh1dGFNGJyhyDjUr8fM7RzAHjD9CfnY8"},"B":{"confirmed":"2018-11-23T00:57:33.354560559Z",
"proposed":{"confirmed":"2018-11-23T00:57:33.354288922Z",
"proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"voting_basis":{"round":0,"height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":2},"transactions":null,"proposer_transaction":{"H":{"version":"1",
"created":"2018-11-23T00:57:33.353805821Z",
"signature":"bTGdFEHjWvA6ZpmX89ERn1txZg3y5PxizeWBKusHomxgmi1awrmwjZv3rLeLhemsmTK5LU7WjDdKMpPtxKfqQnz"},"B":{"source":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"fee":"0",
"sequence_id":0,"operations":[{"H":{"type":"collect-tx-fee"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"0",
"txs":0,"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}},{"H":{"type":"inflation"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"100000000000", "initial_balance":"1000000000000000000",
"ratio":"0.00000010000000000",
"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}}]}}},"source":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"state":1,"vote":"YES", "reason":null}},"caller":"checker.go:331",
"from":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"isMine":false,"lvl":"dbug", "module":"noderunner", "msg":"ballot voted",
"new":true,"node":"GARA.FANE",
"proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"state":"INIT", "t":"2018-11-23T00:57:33.375310818Z", "vote":"YES",
"votingBasis":{"round":0,"height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":2}}
```

```
{"ballot":"F8F9mu4Wx3st9rdoqS57sMy9t5KcgseTBWf972NLqf9P",
"block":{"version":0,"prev_block_hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"transactions_root":"4kqjnWe38toJimMKimJQU5C4vMvQSLizaJ2R2UUoqxRn",
"proposed_time":"2018-11-23T00:57:33.354288922Z",
"height":2,"total-txs":2,"total-ops":4,"transactions":null,"proposer_transaction":"FN6gCZgfD2dwXPty1r7f9bsBFRp2TWusDJ9S7GEkxtGp",
"hash":"GWWGdvaMB8XFBkHqUsTT7mfD7xR49FSEkyQAkjJp3vBJ",
"proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"round":0,"confirmed":"2018-11-23T00:57:33.397593325Z"},"caller":"finish_ballot.go:91",
"from":"GCWNAGLUM5OVIME7Y2Y6PGIRK6NUQZHDSKU7CXOJXKC62RPRFD24C5Z5",
"isMine":false,"lvl":"dbug", "module":"noderunner", "msg":"NewBlock created",
"node":"GARA.FANE",
"proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ",
"state":"ACCEPT", "t":"2018-11-23T00:57:33.397648575Z", "vote":"YES",
"votingBasis":{"round":0,"height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":2}}

{"caller":"finish_ballot.go:92","confirmed":"2018-11-23T00:57:33.397593325Z","height":2,"lvl":"info","module":"noderunner","msg":"NewBlock
created","node":"GARA.FANE","proposer":"GAEIGUV2RBOQ4YNKKDWTCNAQ7ZBZYO3P6RMJ5FJ6VUUZHN7UG3JKZEMJ","round":0,"t":"2018-11-23T00:57:33.397690216Z","total-ops":4,"total-txs":2}

{"caller":"http2_network_log.go:99",
"content-length":1557,"content-type":"application/json",
"headers":{},"host":"172.31.25.219:12000",
"id":"aad3d9fb-deef-4b4b-ac92-94a8e169d6d0", "lvl":"dbug", "method":"POST",
"module":"http", "msg":"request", "node":"GARA.FANE", "proto":"HTTP/2.0",
"referer":"", "remote":"172.31.22.130:48340",
"t":"2018-11-23T00:57:33.398042344Z", "uri":"/node/ballot",
"user-agent":"v-GAEI.G3JK"}
```

```
{"ballot":{"H":{"version":"1",
"hash":"2TGkX9AXpoPVhzuy7r6pNxfSUTto6yZRn68RMi8Z1DTh",
"signature":"5k23yR7xTG4btEKZp11r38NG9iogBCn3UUuLEAdeJGAnyjganhqJKRdK7dp3MyoQ6UUg43DLmmBx4pgUZNt13zjC",
"proposer_signature":"hUVBzcCDUqzHukaqJqNtbTzoi4pWP24vHWtv535iGeW67WQEfrP5jLrzH8x1UXqu2KdFgUMrgp1QHZ8dRUyLMDd"},"B":{"confirmed":"2018-11-24T07:38:26.983134696Z",
"proposed":{"confirmed":"2018-11-24T07:38:26.979901939Z",
"proposer":"GAMEQCFGCLRKNOXXIEIKDS6GRQHUWAGOS7QEW5CLYF3XBLSUXDHNR6DA",
"voting_basis":{"round":0,"height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":2},"transactions":null,"proposer_transaction":{"H":{"version":"1",
"created":"2018-11-24T07:38:26.979457338Z",
"signature":"5LuQ6gjF79Wu3nh7RLFL5T3NZq87WXfRW7d7taFG3hrq5rznZiqvdR58iNxfc7bzkLPvBGZhgjpPFgaYaYSe8FyY"},"B":{"source":"GAMEQCFGCLRKNOXXIEIKDS6GRQHUWAGOS7QEW5CLYF3XBLSUXDHNR6DA",
"fee":"0",
"sequence_id":0,"operations":[{"H":{"type":"collect-tx-fee"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"0",
"txs":0,"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}},{"H":{"type":"inflation"},"B":{"target":"GAIAM4KZWZWQKHALG6YFNPWRRA7JHGCYZ4KQ4RDILG4MGWX7MQA6IZTB",
"amount":"100000000000", "initial_balance":"1000000000000000000",
"ratio":"0.00000010000000000",
"block-height":1,"block-hash":"EaSw9qJeQQtiqsUe4f3Ri134oCQAmNRWcto8Q6CzC3p9",
"total-txs":1,"total-ops":0}}]}}},"source":"GCNGYDIOVCQSKLIM57AGGUS4DL5IFJL7MQRR6LHSFJB22ABONCHGSBX7",
"state":2,"vote":"YES", "reason":null}},"caller":"isaac.go:176", "lvl":"dbug",
"module":"consensus", "msg":"CanGetVotingResult", "node":"GCNG.NCHG",
"t":"2018-11-24T07:38:26.984696842Z"}
```

These messages are too huge and too hard to read.

### Solution

This PR only tries to clean up the log messages and tries to make log to be meaningful.

### CODE REVIEW

**Comment by Geod24** [comment]
@spikeekips : I was considering this issue yesterday. Do you think we could have a "HumanString" method that gives a human-readable representation ? That will be easier than listing the fields.

**Comment by spikeekips** [comment]
@Geod24 It makes sense for info level message.

The debug message unlike others can not be easily readable in place because it has necessary internal data. The debug message should be handled by parsing process, so in our production, `--log-format=json` will be helpful, it can be easily parsed and already structured. :)

File: lib/node/runner/checker_message.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ func PushIntoTransactionPoolFromNode(c common.Checker, args ...interface{}) erro
 		return err
 	}
 
-	checker.Log.Debug("push transaction into TransactionPool from node", "transaction", tx)
+	checker.Log.Debug("push transaction into TransactionPool from node")
```
Comment (anarcher) [inline lib/node/runner/checker_message.go]: How about `from node","tx",tx.Hash)`?

Comment (spikeekips) [reply lib/node/runner/checker_message.go]: @anarcher This ` checker.Log` already have, https://github.com/bosnet/sebak/blob/c3199102056466cbad966bcfbc6de3f08f6b2428/lib/node/runner/checker_message.go#L58 .

**Pull request merged**